lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -52,26 +52,27 @@
        configurations conforms to task arguments.'
     - 'If C(state) is set to C(absent) and virtual machine exists, then the specified virtual machine
       is removed with its associated components.'
-    - 'If C(state) is set to one of the following C(poweredon), C(poweredoff), C(present), C(restarted), C(suspended)
-      and virtual machine does not exists, then virtual machine is deployed with given parameters.'
-    - 'If C(state) is set to C(poweredon) and virtual machine exists with powerstate other than powered on,
+    - 'If C(state) is set to one of the following C(poweredon), C(powered-on), C(poweredoff), C(powered-off),
+      C(present), C(restarted), C(suspended) and virtual machine does not exists, virtual machine is deployed with the given parameters.'
+    - 'If C(state) is set to C(poweredon) or C(powered-on) and virtual machine exists with powerstate other than powered on,
       then the specified virtual machine is powered on.'
-    - 'If C(state) is set to C(poweredoff) and virtual machine exists with powerstate other than powered off,
+    - 'If C(state) is set to C(poweredoff) or C(powered-off) and virtual machine exists with powerstate other than powered off,
       then the specified virtual machine is powered off.'
     - 'If C(state) is set to C(restarted) and virtual machine exists, then the virtual machine is restarted.'
     - 'If C(state) is set to C(suspended) and virtual machine exists, then the virtual machine is set to suspended mode.'
-    - 'If C(state) is set to C(shutdownguest) and virtual machine exists, then the virtual machine is shutdown.'
-    - 'If C(state) is set to C(rebootguest) and virtual machine exists, then the virtual machine is rebooted.'
+    - 'If C(state) is set to C(shutdownguest) or C(shutdown-guest) and virtual machine exists, then the virtual machine is shutdown.'
+    - 'If C(state) is set to C(rebootguest) or C(reboot-guest) and virtual machine exists, then the virtual machine is rebooted.'
+    - 'Powerstate C(powered-on) and C(powered-off) is added in version 2.10.'
```

Comment:
```suggestion
    - 'Powerstate C(powered-on) and C(powered-off) is added in version 2.10.'
    -
```
I think to add the same for ```shutdown-guest``` and ```reboot-guest``` would be good